index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Leehaney</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <div><p id="logo">Leehaney George</p></div>
+      <div><img id="menu" src="./assets/Icon - Menu.png" alt="Menu bar" /></div>
+    </nav>
+    <section>
+      <div id="intro">
+        <h1 id="primary">Hey There. <br />I'm Lee</h1>
+      </div>
+      <p id="texttwo">I'am a Software Developer</p>
+      <p id="textthree">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div id="socialmedia">
+        <a class="icons" href="https://github.com/leehaney254"
+          ><img src="./assets/github.png" alt="Github link"
+        /></a>
+        <a
+          class="icons"
+          href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
+          ><img src="./assets/Linkedin.png" alt="Linkedin icon"
+        /></a>
+        <a
+          class="icons"
+          href="https://app.slack.com/client/T47CT8XPG/D0487PRDM7F/rimeto_profile/U03T2ES9771"
+          ><img src="./assets/slack.png" alt="Slack icon"
+        /></a>
+        <a class="icons" href="https://twitter.com/Lee06785586"
+          ><img src="./assets/twitter.png" alt="Twitter icon"
+        /></a>
+        <a class="icons" href="georgeleehaney@gmail.com"
+          ><img src="./assets/gmail.png" alt="gmail icon"
+        /></a>
+      </div>
+      <img id="decor" src="./assets/Illustration.png" alt="decorations" />
```

Comment:
- [ ] Kindly make sure all background images are inserted in the browser by using CSS properties. Please use all CSS 
 `background` properties to insert this image in your browser instead of inserting it as a static image using HTML.

Reply:
Hello! Kindly check. I have made the requested changes. Thank you.:smiley: